github/provider_test.go

```diff
@@ -40,301 +34,197 @@ func init() {
+			},
+		})
 
-func configProviderOwner(owner string) string {
-	return fmt.Sprintf(`
-provider "github" {
-    owner = "%s"
-}
-`, owner)
-}
+	})
 
-var configProviderOwnerEmpty = `provider "github" {}`
-
-const userResponseBody = `{
-  "login": "hashibot",
-  "id": 1,
-  "node_id": "MDQ6VXNlcjE=",
-  "avatar_url": "https://github.com/images/error/octocat_happy.gif",
-  "gravatar_id": "",
-  "url": "https://api.github.com/users/octocat",
-  "html_url": "https://github.com/octocat",
-  "followers_url": "https://api.github.com/users/octocat/followers",
-  "following_url": "https://api.github.com/users/octocat/following{/other_user}",
-  "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
-  "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
-  "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
-  "organizations_url": "https://api.github.com/users/octocat/orgs",
-  "repos_url": "https://api.github.com/users/octocat/repos",
-  "events_url": "https://api.github.com/users/octocat/events{/privacy}",
-  "received_events_url": "https://api.github.com/users/octocat/received_events",
-  "type": "User",
-  "site_admin": false,
-  "name": "HashiBot",
-  "company": "GitHub",
-  "blog": "https://github.com/blog",
-  "location": "San Francisco",
-  "email": "octocat@github.com",
-  "hireable": false,
-  "bio": "There once was...",
-  "public_repos": 2,
-  "public_gists": 1,
-  "followers": 20,
-  "following": 0,
-  "created_at": "2008-01-14T04:33:35Z",
-  "updated_at": "2008-01-14T04:33:35Z"
-}`
-
-const gpgKeysResponseBody = `[
-  {
-    "id": 3,
-    "primary_key_id": null,
-    "key_id": "3262EFF25BA0D270",
-    "public_key": "xsBNBFayYZ...",
-    "emails": [
-      {
-        "email": "mastahyeti@users.noreply.github.com",
-        "verified": true
-      }
-    ],
-    "subkeys": [
-      {
-        "id": 4,
-        "primary_key_id": 3,
-        "key_id": "4A595D4C72EE49C7",
-        "public_key": "zsBNBFayYZ...",
-        "emails": [
-        ],
-        "subkeys": [
-        ],
-        "can_sign": false,
-        "can_encrypt_comms": true,
-        "can_encrypt_storage": true,
-        "can_certify": false,
-        "created_at": "2016-03-24T11:31:04-06:00",
-        "expires_at": null
-      }
-    ],
-    "can_sign": true,
-    "can_encrypt_comms": false,
-    "can_encrypt_storage": false,
-    "can_certify": true,
-    "created_at": "2016-03-24T11:31:04-06:00",
-    "expires_at": null
-  }
-]`
-
-const keysResponseBody = `[
-  {
-    "id": 1,
-    "key": "ssh-rsa AAA..."
-  }
-]`
-
-func orgResponseBody(port string) string {
-	url := fmt.Sprintf(`https://localhost:%s/v3/orgs/%s`, port, testOwner)
-	return fmt.Sprintf(`
-{
-	"login": "%s",
-	"url" : "%s",
-	"repos_url": "%s/repos"
-}
-`, testOwner, url, url)
+	t.Run("can be configured with an organization account", func(t *testing.T) {
+
+		organizationConfiguration := fmt.Sprintf(`
+			provider "github" {
+				organization = "%s"
```

Comment:
Nit: could use format with index `"%[1]s"`

Reply:
❤️ did not know this magic existed.  added in a recent commit.